certified-connectors/Polaris PSA/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2662 @@
+          "type": "object",
+          "properties": {
+            "timesheet": {
+              "type": "object",
+              "properties": {
+                "displayText": {
+                  "type": "string",
+                  "description": "displayText"
+                },
+                "uri": {
+                  "type": "string",
+                  "description": "uri"
+                },
+                "slug": {
+                  "type": "string",
+                  "description": "slug"
+                }
+              },
+              "description": "timesheet"
+            },
+            "approvalStatusUri": {
+              "type": "string",
+              "description": "approvalStatusUri"
+            },
+            "authority": {
+              "type": "object",
+              "properties": {
+                "actingUser": {
+                  "type": "object",
+                  "properties": {
+                    "displayText": {
+                      "type": "string",
+                      "description": "user display Text"
+                    },
+                    "loginName": {
+                      "type": "string",
+                      "description": "user login Name"
+                    },
+                    "uri": {
+                      "type": "string",
+                      "description": "uri"
+                    },
+                    "slug": {
+                      "type": "string",
+                      "description": "slug"
+                    }
+                  },
+                  "description": "actingUser"
+                },
+                "displayText": {
+                  "type": "string",
+                  "description": "displayText"
+                },
+                "actingForUser": {
+                  "type": "string",
+                  "description": "actingForUser"
+                },
+                "authorityType": {
+                  "type": "object",
+                  "properties": {
+                    "displayText": {
+                      "type": "string",
+                      "description": "displayText"
+                    },
+                    "uri": {
+                      "type": "string",
+                      "description": "uri"
+                    }
+                  },
+                  "description": "authorityType"
+                }
+              },
+              "description": "authority"
+            },
+            "timesheetStatusUri": {
+              "type": "string",
+              "description": "timesheetStatusUri"
+            }
+          }
+        },
+        "description": "the response for webhook subscription"
+      },
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/WebhookSubscriptionResponse"
+            }
+          }
+        },
+        "summary": "WebhookSubscriptionsRestAPI",
```

Comment:
@repl-anand-vikram It looks like an operation Id rather than a summary value, please provide a valid summary value.